source/index.html

```diff
@@ -1,12 +1,147 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
+<html class="page" lang="ru">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS, уровень 2</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body class="page__body">
+  <div class="page-header__wrapper">
+    <header class="page-header">
+      <a href="" class="page-header__logo">
+        <img src="https://via.placeholder.com/191x66" alt="Логотип Cat Energy, ссылка на Главную страницу">
+      </a>
+
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--active">
+            <a href="">Главная</a>
+          </li>
+          <li class="site-list__item">
+            <a href="catalog.html">Каталог продукции</a>
+          </li>
+          <li class="site-list__item">
+            <a href="">Подбор программы</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+  </div>
+
+  <main class="page-main">
+    <section class="programme">
+      <h1 class="programme__title">Функциональное питание для котов</h1>
+      <p class="programme__intro">Занялся собой? Займись котом!</p>
+      <img src="https://via.placeholder.com/280x270"
+        alt="Упаковка кошачьего корма 'Cat Energy Pro' с курицей, 500 грамм. Натуральность корма: 0%">
+      <a class="programme__link" href="form.html">Подобрать программу</a>
+    </section>
+
+    <section class="catalogs">
+      <ul class="catalogs__list">
+        <li class="catalogs__item catalogs-item">
+          <h2 class="catalogs-item__title">Похудение</h2>
+          <p class="catalogs-item__intro">
+            Ваш кот весит больше собаки и
+            почти утратил способность лазить
+            по деревьям? Пора на диету! Cat
+            Energy Slim поможет вашему
+            питомцу сбросить лишний вес.
+          </p>
+          <a class="catalogs-item__link" href="">Каталог slim</a>
+        </li>
+
+        <li class="catalogs__item catalogs-item">
+          <h2 class="catalogs-item__title">Набор массы</h2>
+          <p class="catalogs-item__intro">
+            Заработать авторитет среди
+            дворовых котов и даже собак?
+            Серия Cat Energy Pro поможет
+            вашему коту нарастить
+            необходимые мышцы!
+          </p>
+          <a class="catalogs-item__link" href="">Каталог Pro</a>
+        </li>
+      </ul>
+    </section>
+
+    <section class="features">
+      <h2 class="features__title">Как это работает</h2>
+      <ul class="features__list">
+        <li class="features__item">
```

Comment:
Тут можно замиксовать feature-item и от него идти вниз

Reply:
```suggestion
        <li class="features__item feature-item">
```
Типа такого, и так для каждого `li` из `features_list`?

Reply:
Да. Это решает проблему слишком большой вложенности.